resources/views/livewire/server/form.blade.php

```diff
@@ -10,12 +10,17 @@
                 </x-new-modal>
             @else
                 <x-forms.button type="submit">Save</x-forms.button>
+                <x-new-modal buttonTitle="Revalidate" title="Revalidate" action="revalidate">
+                    This will revalidate the server, install / update Docker Engine, Docker Compose and all related
+                    configuration. It will also restart Docker Engine, so your running containers will be unreachable
+                    for the time being.
+                </x-new-modal>
```

Comment:
You added the option to revalidate a server only for secondary servers. Can it also be used for primary servers (the server Coolify is running on)? For me `coolify-proxy` doesn't start on server-startup or when I click "Restart proxy", so maybe re-validating everything can fix it?